resources/lang/en/secretariat.php

```diff
@@ -6,6 +6,8 @@
     'semester_status' => 'Semester status',
     'semester_status_info' => 'Active - You will be an active member of the Collegium<br>Passive - You will be on a foregin semester or are passive in the university<br>Deactivated - You will not be an active member of the Collegium',
     'status' => 'Status',
+    'status_set' => 'Your status for this semester has already been set, it is',
+    'status_statement_not_available' => 'The status update form is not available. You have already set your status and the deadline to set it has passed.',
```

Comment:
I think it's an 'or', not an 'and'

Reply:
the status form is open if the deadline has not passed AND the status is not set :D

Reply:
Oh, you did not make the check like this.

Reply:
Would not it make more sense?

Reply:
It does not make too much difference as everyone's status will be set to DEACTIVATED if they didn't set it themselves. I think it is okay to allow people to set their status multiple times before the deadline, for example if they messed it up the first time.

Reply:
Okay